lib/shopify_cli/services/app/create/rails_service.rb

```diff
@@ -148,9 +155,8 @@ def check_yarn
           end
 
           def build(name, db)
-            context.abort(context.message("core.app.create.rails.error.install_failure",
-              "rails")) unless install_gem("rails",
-                "<6.1")
+            context.abort(context.message("core.app.create.rails.error.install_failure", "rails")) unless
```

Comment:
Not a criticism of your work, I know it existed before, I just want to point out that this `unless` carrying over to the next line is a weird pattern.